components/Header/Artist.tsx

```diff
@@ -29,6 +29,12 @@ const Artist = ({
       if (selectedArtist.account_id !== artist?.account_id) push("/");
     }
     setSelectedArtist(artist);
+    
+    // Route to the /new page when an artist is selected from the sidebar
+    // Only redirect if we're not already on the /new page
+    if (!pathname.includes("/new")) {
+      push("/new");
+    }
```

Comment:
Why are we redirecting onClick of an artist?
- If I am on `/posts` I want to click through my active artists to see the posts for each active artist. If you redirect me, I have more clicks and added frustration.